static/js/admin.js

```diff
@@ -302,7 +307,7 @@ function getRightSideGradingDiv(element, acid, studentId) {
             var file_div_template = '<pre id="file_div_template" style = "display:none;">template text</pre>;'
             var index;
             for (index = 0; index < data.file_includes.length; index += 1) {
-                if (jQuery('#' + data.file_includes[index].acid).length == 0) {
+                if (jQuery('#' + data.file_includes[index].acid,rightDiv).length == 0) {
```

Comment:
Is this correct?  What does the ,rightDiv part do?

Reply:
,rightDiv requires that the element be a child of rightDiv; there are now multiple rightDivs, one for each problem to grade, and this is easier than giving all the children of each rightDiv unique names.

Reply:
gotcha, I've not used that syntax with jQuery before.